cmd/launcher/launcher.go

```diff
@@ -182,11 +181,11 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 
 	var controlService *control.ControlService
 	var runner *desktopRunner.DesktopUsersProcessesRunner
-	if (opts.KolideServerURL == "k2device-preprod.kolide.com" || opts.KolideServerURL == "localhost:3443") && runtime.GOOS != "linux" {
+	if opts.KolideServerURL == "k2device-preprod.kolide.com" || opts.KolideServerURL == "localhost:3443" {
```

Comment:
This also gates running the desktop process. Do we want to update so that it only gates the control server, and we always run the desktop process now?

Reply:
I _think_ we still want desktop to be internal for now, until we make it more useful and can effectively enable it only for beta users, for example.